libs/entry-components/configure-theming.md

```diff
@@ -0,0 +1,435 @@
+
+- `size` determines the font size of the dialog title.
+
+```scss
+$custom-theme: (
+ dialogs: (
+  title: (
+  size: 20px
+ )
+)
+);
+```
+
+## 2. Native Angular Material Configuration
+
+With this approach we configure every little piece in our entry components. Angular Material defines a predefined color palette that you can use in your application. The palette includes a range of primary, accent, and warn colors.
+Check these list for more info
+
+- [Theming Guide](https://material.angular.io/guide/theming)
+- [Typography Guide](https://material.angular.io/guide/typography)
+
+Bellow is example:
+
+```scss
+$typo: (headline-1: (font-size: 96px, line-height: 96px, font-weight: 300, font-family: 'roboto, sans-serif', letter-spacing: -.015625em),
+headline-2: (font-size: 60px, line-height: 60px, font-weight: 300, font-family: 'roboto, sans-serif', letter-spacing: -.0083333333em), 
+headline-3: (font-size: 48px, line-height: 50px, font-weight: 400, font-family: 'roboto, sans-serif', letter-spacing: normal),
+headline-4: (font-size: 34px, line-height: 40px, font-weight: 400, font-family: 'roboto, sans-serif', letter-spacing: .0073529412em), 
+headline-5: (font-size: 32px, line-height: 32px, font-weight: 400, font-family: 'roboto, sans-serif', letter-spacing: normal), 
+headline-6: (font-size: 32px, line-height: 32px, font-weight: 500, font-family: 'roboto, sans-serif', letter-spacing: .0125em),
+subtitle-1: (font-size: 32px, line-height: 28px, font-weight: 400, font-family: 'roboto, sans-serif', letter-spacing: .009375em),
+subtitle-2: (font-size: 32px, line-height: 22px, font-weight: 500, font-family: 'roboto, sans-serif', letter-spacing: .0071428571em),
+body-1: (font-size: 16px, line-height: 24px, font-weight: 400, font-family: 'roboto, sans-serif', letter-spacing: .03125em), 
+body-2: (font-size: 14px, line-height: 20px, font-weight: 400, font-family: 'roboto, sans-serif', letter-spacing: .0178571429em), 
+caption: (font-size: 32px, line-height: 20px, font-weight: 400, font-family: 'roboto, sans-serif', letter-spacing: .0333333333em),
+button: (font-size: 24px, line-height: 36px, font-weight: 500, font-family: 'roboto, sans-serif', letter-spacing: .0892857143em), 
+overline: (font-size: 12px, line-height: 32px, font-weight: 500, font-family: 'roboto, sans-serif', letter-spacing: .1666666667em), 
+'font-family': 'roboto, sans-serif');
+$prim: (
+ 50 : #E6E6E6,
+ 100 : #C0C0C0,
+ 200 : #2B95DB,
+ 300 : #7FE2F3,
+ 400 : #B5D117,
+ 500 : hsl(78 78% 47%),
+ 600 : #C953EC,
+ 700 : #1A81C5,
+ 800 : #209E94,
+ 900 : #2FC955,
+ A100 : #209E94,
+ A200 : #1A81C5,
+ A400 : #C953EC,
+ A700 : #B5D117,
+ contrast: (
+  50 : #000,
+  100 : #000,
+  200 : #000,
+  300 : #FFF,
+  400 : #FFF,
+  500 : #FFF,
+  600 : #FFF,
+  700 : #FFF,
+  800 : #FFF,
+  900 : #FFF,
+  A100 : #000,
+  A200 : #FFF,
+  A400 : #FFF,
+  A700 : #FFF,
+ )
+);
+
+$custom-theme: (
+ general: (
+  colors: (
+   primary-theme: $prim,
```

Comment:
No abbreviations in code examples, please! This code must be representative for the rest of our company and open source community.